dataland-frontend/src/utils/SearchCompaniesForFrameworkDataPageDataRequester.ts

```diff
@@ -5,69 +5,22 @@
 
 import { ApiClientProvider } from "@/services/ApiClients";
 import {
-  type StoredCompany,
-  type CompanyInformation,
-  type DataMetaInformation,
+  BasicCompanyInformation,
   type DataTypeEnum,
-  QaStatus,
-  IdentifierType,
 } from "@clients/backend";
 import type Keycloak from "keycloak-js";
 import { ARRAY_OF_FRAMEWORKS_WITH_VIEW_PAGE } from "@/utils/Constants";
 
-export interface DataSearchStoredCompany {
-  companyName: string;
-  companyInformation: CompanyInformation;
-  companyId: string;
-  permId: string;
-  dataRegisteredByDataland: Array<DataMetaInformation>;
-}
-
 export interface FrameworkDataSearchFilterInterface {
   companyNameFilter: string;
   frameworkFilter: Array<DataTypeEnum>;
   countryCodeFilter: Array<string>;
   sectorFilter: Array<string>;
 }
 
-/**
- * Retrieve the value of the Perm Id of a company. Throws an exception if no perm id is found
```

Comment:
Gone, because the return of the modified endpoint gives those information directly?

Reply:
yes

Reply:
thank you